src/crypto/ring.rs

```diff
@@ -79,3 +86,15 @@ impl TryFrom<DigestAlgorithm> for &'static digest::Algorithm {
         }
     }
 }
+
+impl TryFrom<DigestAlgorithm> for hmac::Algorithm {
```

Comment:
Do we still need the `TryFrom` impl bellow?

Reply:
Yes, both are used.